### ISSUE TITLE
notify_original_recipients option

### ISSUE THREAD

**Issue by hedenface** [NagiosEnterprises/nagioscore#291]
If you have a host with contact A defined, and a down escalation defined for that host with contacts B and C, and then a recovery escalation defined for that host with contacts D and E - by definition of how the escalation groups work, only contacts D and E will be notified of the recovery, regardless of whether or not host down notifications were sent to contacts A B or C.

It would be nice to have a feature to force send recovery notifications to those contacts which have already received notifications.

**Comment by ericloyd** [comment]
Won't contact A also receive the recovery (assuming all other things are equal) as part of the original notification pathway?

**Comment by hedenface** [comment]
No. You can test this by creating a recovery escalation for a host with a different contact than what is defined on the host. Get the host to a down state and notice that the notifications are sent to the host's contact.

Then get the host to a recovery state, and notice that only the host defined in the recovery escalation is notified.

The escalations override the contacts and contact groups for the hosts and services they are defined for during the specified period (first - last). If I said that right.

**Comment by hedenface** [comment]
Will review at the next meeting.
